src/core/symbology-ng/qgsrulebasedrendererv2.cpp

```diff
@@ -142,6 +153,14 @@ void QgsRuleBasedRendererV2::Rule::updateElseRules()
   }
 }
 
+void QgsRuleBasedRendererV2::Rule::setIsElse( bool iselse )
```

Comment:
Oh, I just noticed it was already wrong in the *.h file. So we better leave this for QGIS 3, I guess.